### ISSUE TITLE
[UD] preferred_type is not respected by dashboard

### ISSUE THREAD

**Issue by sleshchenko** [eclipse-che/che#19069]
### Describe the bug
[UD] preferred_type is not respected by dashboad

### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce
1. Deploy che with non-default storage type as preferred:
```
chectl server:deploy --installer=operator --platform=openshift --che-operator-cr-patch-yaml=/home/sleshche/che-custom-che-server.yaml
spec:
  server:
    cheImagePullPolicy: Always
    customCheProperties:
      CHE_WORKSPACE_STORAGE_AVAILABLE__TYPES: persistent,ephemeral,async
      CHE_WORKSPACE_STORAGE_PREFERRED__TYPE: ephemeral
```
2. Check that it's used on Get Started
![Screenshot_20210215_142852](https://user-images.githubusercontent.com/5887312/107947283-dbe81980-6f9a-11eb-8b93-f69eaaac5945.png)

3. Check that it's used on Create Custom Workspace
![Screenshot_20210215_142903](https://user-images.githubusercontent.com/5887312/107947298-e0accd80-6f9a-11eb-904f-f6f0884d4ea4.png)

4. Create that it's used on Factory flow
![Screenshot_20210215_143120](https://user-images.githubusercontent.com/5887312/107947310-e4405480-6f9a-11eb-9d69-c6e150e8e7f2.png)

**Comment by sleshchenko** [comment]
Get started page will be fixed in 7.28.
Not timeline for other cases yet

**Comment by sleshchenko** [comment]
tested and after https://github.com/eclipse/che-dashboard/pull/187 all of the described scenarios work as expected - preferable type is applied:
- for factory flow - only when devfile does not have specified storage attributes
- for create custom workspace(templates + load devfile) - always, storage attributes are always overridden. A dedicated issue will be created to make custom workspace page respect devfile storage attributes, and apply default only when they are missing.
